<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindRequirements</class>
 <widget class="QDialog" name="FindRequirements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="p_groupBox_Find">
     <property name="title">
      <string>Requirements data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="p_lbl_Position">
        <property name="text">
         <string>Position</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="p_comboBox_Positions"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="p_lbl_StartDate">
        <property name="text">
         <string>Start date</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_StartDate"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="p_lbl_EndDate">
        <property name="text">
         <string>End date</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_EndDate"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="p_groupBox_Result">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="p_treeWidget_Requirements">
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked</set>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">4</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">5</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
